app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -57,7 +57,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onStart() {
         super.onStart();
 
-        loginFragmentViewModel.getLoginStatus().observe(this, this::handleIntent);
+        loginFragmentViewModel.getLoginStatus().observe(this, (loginStatus) -> {
```

Comment:
Revert as previous behaviour was desirable

Reply:
We'd then have to use the `handleIntent()` method which takes an unused boolean parameter, and we'll have to suppress its PMD warning it using `"unused" parameter`.

Reply:
Remove the boolean param from there

Reply:
The method `handleIntent()` referenced by the observer needs to have the Boolean parameter because it's observing LiveData<Boolean>. I get error if I remove it.
![screenshot from 2018-06-01 07-31-32](https://user-images.githubusercontent.com/35009811/40817043-de5b131e-656d-11e8-8bcf-1939bc500e13.png)